### ISSUE TITLE
When Elk is opened as an app in iOS, the refresh gesture does not work anymore

### ISSUE THREAD

**Issue by behrangsa** [elk-zone/elk#1712]
Demo:

https://user-images.githubusercontent.com/18451/218245113-3f638eb9-1831-4bd2-b959-7761c798064b.MOV

**Comment by ctbk** [comment]
I would like to add something to this issue: the refresh gesture doesn't work _and_ the link "There are X new posts" never appears. 
The only way to get a refresh is to kill the "app" and restart it.

**Comment by phpmacher** [comment]
Unfortunately this is still the case

**Comment by santisbon** [comment]
This happens on both iOS and iPadOS (Safari). On macOS (Chrome) it works fine.

**Comment by threkk** [comment]
Could it be possible to add a reload button (for the timeline or the whole application, if needed) somewhere in the UI? This would solve the issue.

**Comment by behrangsa** [comment]
I'll have a crack at this next week. I haven't used Vue.js before so I shall be using Copilot and OpenAI extensively to help me with that.

**Comment by miemo** [comment]
Yup, having to keep on killing and restarting the PWA on iOS/iPadOS just to load new posts is a major bottleneck to otherwise great alpha of a product.

**Comment by phoerious** [comment]
It's been a while. Is this still being worked on?

**Comment by luomavaltteri** [comment]
Wondering the same thing. This is basically the only issue that keeps me going back to native apps at the moment on iOS.

**Comment by jan-xyz** [comment]
The gesture to pull down to refresh is a native feature of Safari and not Elk. When turning it into a PWA you basically lose this feature because the PWA framework doesn't have this

**Comment by phoerious** [comment]
The "X new posts" indicator at the top is super unreliable and even if it appears, it often doesn't load all new posts. Right now, the only way to refresh the feed is to force-close the app. I would consider this a breaking bug.

**Comment by Morse6690** [comment]
Still needing to force close and restart to get full refresh of feeds. 
Otherwise, this is an awesome client!
